src/deploy/hosting/convertConfig.ts

```diff
@@ -58,8 +58,8 @@ export function convertConfig(config?: HostingConfig): { [k: string]: any } {
         vRewrite.path = rewrite.destination;
       } else if ("function" in rewrite) {
         vRewrite.function = rewrite.function;
-        if (rewrite.function_region) {
-          vRewrite.function_region = rewrite.function_region;
+        if (rewrite.region) {
+          vRewrite.function_region = rewrite.region;
         } else {
           vRewrite.function_region = "us-central1";
```

Comment:
If DynamicLinks is cammelcase, I think function_region should be here, too

Reply:
Ah yes, this is true. I do think this should work too